x-pack/plugins/security_solution/public/detections/components/rules/step_define_rule/index.tsx

```diff
@@ -899,68 +903,62 @@ const StepDefineRuleComponent: FC<StepDefineRuleProps> = ({
             </>
           )}
 
-          <EuiSpacer size="l" />
-          <EuiAccordion
-            data-test-subj="alertSuppressionAccordion"
-            id="alertSuppressionAccordion"
-            buttonContent={i18n.ALERT_SUPPRESSION_ACCORDION_BUTTON}
+          <RuleTypeEuiFormRow
+            $isVisible={isQueryRule(ruleType)}
+            data-test-subj="alertSuppressionInput"
```

Comment:
ux nit: May want to consider hiding the suppression configuration UI until a field is selected, as it's in a disabled state until the user selects a field, which adds extra information to an already well populated form

current:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236561568-43a4f005-becb-413b-8615-baca6535a0fb.png">

hidden w/o selection:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236561687-d5f4d01f-1c6c-449e-9fc6-60e1f573fa7a.png">

visible w/ selection:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236561798-4686c785-3b9b-4d8f-9696-669ca4c345f8.png">

---

We do something similar w/ the expanded Severity/Risk Score override UI:

<img width="300" alt="image" src="https://user-images.githubusercontent.com/2946766/236561930-af1437d2-f9bd-494f-bca9-9395821d28c3.png"> <img width="275" alt="image" src="https://user-images.githubusercontent.com/2946766/236561986-b7462dcf-d53f-4a23-83dd-e5c090d7f716.png">

Reply:
Thanks for suggestions with  different approaches on UI.

We, actually, considered each of them + accordion, to wrap all suppression settings, on Advanced Correlation Workgroup meetings with Product and UI teams.
https://github.com/elastic/kibana/issues/150101#issuecomment-1520223000
https://github.com/elastic/kibana/issues/150101#issuecomment-1515156685

In the end, we decided to stick to current implementation: just intend suppression settings field on rules form.